src/invoices/InvoicesList/Invoice/utils.js

```diff
@@ -35,3 +35,20 @@ export const createInvoiceLineFromPOL = (poLine, invoiceId, vendor) => {
     ...optionalProps,
   };
 };
+
+export const getAlwaysShownAdjustmentsList = (adjustments) => {
```

Comment:
I know you work on this function,

but `filter` should be before `map` and no need 2 `map`